hyp3_gamma/insar/ifm_sentinel.py

```diff
@@ -247,8 +264,8 @@ def make_parameter_file(mydir, parameter_file_name, alooks, rlooks, dem_source):
         f.write('Speckle filtering: off\n')
 
 
-def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, look_flag=False,
-                         los_flag=False, wrapped_flag=False):
+def insar_sentinel_gamma(reference_file, secondary_file, rlooks=20, alooks=4, include_look_vectors=False,
+                         include_los_deformation=False, include_wrapped_phase=False, include_inc_map=False):
```

Comment:
I like the updated parameter names!  I think we use currently `LOS displacement` rather than `LOS deformation` in the README and in Vertex.  I suspect we should use displacement here for consistency?  Or do you prefer we move towards using deformation everywhere?

Reply:
It should be displacement.  I've changed it in my current copy